install_test.sh

```diff
@@ -3,7 +3,12 @@
 set -e
 
 # Lint.
-shellcheck -s sh ./*.sh
+
+# FIXME (caspervonb) was unable to install spellcheck via choco on
+# windows-latest so only run spellcheck if available
+if [ -x "$(command -v spellcheck)" ]; then
```

Comment:
spellcheck != shellcheck

`shellcheck` is available for Windows.

Reply:
Oops! indeed; will install **shell**check via choco in the workflow.